ci/run-check.sh

```diff
@@ -30,11 +30,25 @@ case $TARGET in
     ;;
 
   lint-clippy)
-    cargo clippy --workspace -- -D warnings -A clippy::unnecessary-cast -A clippy::bool-to-int-with-if
+    cargo clippy -- -D warnings -A clippy::unnecessary-cast -A clippy::bool-to-int-with-if
     ;;
   benchmark-check)
     ./scripts/check_benchmarks.sh $RUNTIME
     ;;
   docs-build)
     RUSTDOCFLAGS="-D warnings" cargo doc --all --no-deps
+    ;;
+  subalfred)
+    # Find all child directories containing Cargo.toml files
+    # TODO: Filter by crates found in the workspace
+    dirs=$(find . -name Cargo.toml -print0 | xargs -0 -n1 dirname | sort -u)
+
+    # Execute the command "subalfred check" on each directory
+    for dir in $dirs; do
+      # Avoiding cargo workspace
+      if [[ "$dir" == "." ]]; then
+        continue
+      fi
+      subalfred check features $dir
+    done
```

Comment:
@wischli I finally added it here in order to let it prepared for a future CI job, is it ok for you?